### DISCUSSION TITLE
AD0001 in specific app.json

### DISCUSSION THREAD

**Discussion by SorenBogelund-ECIT** [StefanMaron/BusinessCentral.LinterCop#680] [Q&A]
I have an issue with a specific app.json in a specific repository. The issue does not appear in other app.json files. For some reason 10 warnings are generated for no reason?
![image](https://github.com/StefanMaron/BusinessCentral.LinterCop/assets/169046245/8c486e3f-66fb-4c3a-b3e7-37464f42557e)

**Comment by Arthurvdv** [comment]
https://github.com/StefanMaron/BusinessCentral.LinterCop/issues/682
